sdk/python/feast/pyspark/launcher.py

```diff
@@ -123,22 +129,45 @@ def start_historical_feature_retrieval_job(
     job_id: str,
```

Comment:
i think it would make sense to remove `job_id` here as well

Reply:
Noted. Will take note of this in my follow up PR.

Reply:
Removed the job_id